tutormfe/plugin.py

```diff
@@ -17,12 +17,6 @@
             "name": "account",
             "repository": "https://github.com/edx/frontend-app-account",
             "port": 1997,
-            "env": {
-                "production": {
-                    "COACHING_ENABLED": "",
-                    "ENABLE_DEMOGRAPHICS_COLLECTION": "",
-                },
-            },
```

Comment:
Forgot to mention explicitly: we also need to remove the `PROFILE_MFE_APP` env below, and move it to the lms settings.

Reply:
Can you also change the `edx` Github repos to `openedx`, please?